docs/dagit-screenshot/tox.ini

```diff
@@ -1,15 +0,0 @@
-[tox]
```

Comment:
is this deletion deliberate?

Reply:
this is only for mypy?

Reply:
Yes only `mypy` was run on it. It's an internal utility (glorified script) for taking dagit screenshots that doesn't have any unit tests.